src/ts/containers/Home.tsx

```diff
@@ -1,13 +1,15 @@
+import { browser } from 'webextension-polyfill-ts'
 import * as React from 'react'
 import styled from 'styled-components'
 
 class Home extends React.Component {
 
   render () {
+    let hello = browser.i18n.getMessage('welcome')
```

Comment:
Does that mean we need to call this for all the messages we want to i18n?
Can it be as simple as {t('hello')}  like https://github.com/i18next/react-i18next ?

Reply:
Yes, that's even better. I'll see if I can get this working tonight